#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("debug_mode", bool_t, 0, "Run in debug mode", True)
gen.add("tracking_type", str_t, 0, "Type of tracking", "First Largest")
gen.add("blob_distance_threshold", double_t, 0, "Maximum allowed distance from the previously tracked blob", 800, 0, 1000)

exit(gen.generate("mcr_blob_tracking", "mcr_blob_tracking", "BlobTracking"))